Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM node:current-alpine
 WORKDIR /app
+
 COPY package.json /app
```

Comment:
Setting the WORKDIR above automatically makes COPY CMD RUN use the directory set there.

You can do a `COPY package.json .`